lorem_test.go

```diff
@@ -52,13 +52,15 @@ func TestFakeWord(t *testing.T) {
 
 func TestFakeSentence(t *testing.T) {
 	res := Sentence()
+	fmt.Println(res)
 	if res == "" || !strings.HasSuffix(res, ".") {
 		t.Error("Expected sentence")
 	}
 }
 
 func TestFakeParagraph(t *testing.T) {
 	res := Paragraph()
+	fmt.Println(res)
```

Comment:
can remove this please?

Reply:
During the test, the results were printed to see if they were as expected. When I submitted the PR, I forgot to delete it. 😆

Reply:
> can remove this please?

done

Reply:
I like to use `t.Log()` here.